tests/integration/bond_test.py

```diff
@@ -169,3 +169,48 @@ def test_rollback_for_bond(eth1_up, eth2_up):
 
     current_state_after_apply = netinfo.show()
     assert current_state == current_state_after_apply
+
+
+def test_add_slave_to_bond_without_slaves(eth1_up, setup_remove_bond99):
+    current_bond_state = {
+            INTERFACES: [
+                {
+                    'name': 'bond99',
+                    'type': 'bond',
+                    'state': 'up',
+                    'link-aggregation': {
+                        'mode': 'balance-rr',
+                        'slaves': []
+                    },
+                }
+
+            ]
+        }
+
+    netapplier.apply(current_bond_state)
+
+    desired_state = {
+        INTERFACES: [
+            {
+                'name': 'bond99',
+                'type': 'bond',
+                'state': 'up',
+                'ipv4': {
```

Comment:
This is doing more than what the test name mentions.
Can you please remove the `ipv4` part?

Reply:
Removed